### PR TITLE
added manifest.webapp and 512px icon for Firefox OS

### PR DESCRIPTION
**Pull request by acidicX**
I've added a manifest.webapp and the the 512px icon so that you can "install" (add icon to homescreen) the webapp on Firefox OS devices.

### CODE REVIEW

**Comment by acidicX** [comment]
See issue #1204 for delivery problems

**Comment by engelgabriel** [comment]
Thanks for the help @acidicX 

Do you really need to add the extra image of can you just point to `public/images/logo/512x512.png` on the `manifest.webapp`?

**Comment by acidicX** [comment]
Ah yes, that would work as well. Seems I didn't look long enough! I changed it now.

File: public/manifest.webapp [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "RocketChat",
+    "description": "The Ultimate Open Source WebChat Platform",
+    "launch_path": "/",
+    "icons": {
+        "128": "/images/logo/favicon-128x128.png",
+        "512": "/images/logo/favicon-512x512.png"
```
Comment (engelgabriel) [inline public/manifest.webapp]: Have to change to `"/images/logo/512x512.png"`

Comment (acidicX) [inline public/manifest.webapp]: sorry again, forgot to commit the manifest as well

**Comment by meaku** [comment]
:+1:

**Pull request merged**